docs/FAQ/FAQ-Wallet.md

```diff
@@ -96,7 +96,7 @@ Using different derivation schemes with your extended public key, you can also c
 |Multi-sig P2SH |	2-of-xpub1...-xpub2...-[legacy] |
 
 :::tip
-On top of the xPub extended public key formats shown above, BTCPay Server supports yPub and zPub formats. Please note that these will be converted to xPub once the wallet setup is completed. This has no effect on how you receive or send funds.
```

Comment:
Removed this because it is too likely to confuse users that the xpub doesn't really matter. Although the z/x/yPub doesn't matter, it is quite common for users to just arbitrarily modify their xpub thinking it doesn't really matter. (Examples like issue [#855])

Reply:
This tip was initially put because of the recurring questions about "I put my public key, but it got changed, have I been hacked ?" where a zpub or y pub was added and user didn't know it was converted to xpub.

Reply:
@Zaxounette ok I have added it back that is also a fair point and user question. Thanks!